dev/functional/tests/cypress/page-objects/Institution-objects/InstitutionProfileObject.ts

```diff
@@ -24,7 +28,7 @@ export default class InstitutionProfileObject {
   }
 
   primaryPhoneNumberInputText() {
-    return cy.get("[name='data[primaryPhone]']");
+    return cy.get("[data-cy='primaryPhone']");
   }
 
   institutionWebsiteInputText() {
```

Comment:
I think logic inside line  34, 38,42, 46 can also use `data-cy=`

Reply:
Added data-cy as a selector where elements are getting highlighted. If data-cy as a selector does not get highlighted when selecting any fields, mentioned in attached document.

[selector issues.docx](https://github.com/bcgov/SIMS/files/8780675/selector.issues.docx)

<img width="354" alt="image" src="https://user-images.githubusercontent.com/98920571/170536966-38e2f0f9-2710-4267-8d07-8fb05c5a0492.png">

Reply:
Please schedule a meeting to go over this.